package.json

```diff
@@ -200,6 +195,8 @@
     "@types/node": "^14.14.20 || ^16.0.0",
     "@types/node-cleanup": "^2.1.1",
     "@types/prompts": "2.0.11",
+    "@types/react": "^16",
+    "@types/react-dom": "^16",
```

Comment:
Just to ensure yarn doesn't get 'confused' and installs multiple version of this. We use version 16 in the monorepo.